nilearn/glm/tests/test_dmtx.py

```diff
@@ -117,45 +90,32 @@ def test_cosine_drift():
     # nistats_drifts is placing the constant at the end [:, : - 1]
 
 
-def test_design_matrix0():
+def test_design_matrix_no_experimental_paradigm(frame_times, n_frames):
```

Comment:
but n_frames = len(frame_times), no need to make it explicit.

Reply:
ha indeed, let me remove the unnecessary ones

Reply:
done